python/paddle/vision/__init__.py

```diff
@@ -12,21 +12,74 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import models
 from .models import *
 
-from . import transforms
 from .transforms import *
 
-from . import datasets
 from .datasets import *
 
-from . import image
 from .image import *
 
-from . import ops
 
-__all__ = models.__all__ \
-        + transforms.__all__ \
-        + datasets.__all__ \
-        + image.__all__
+__all__ = [     #noqa
+           'ResNet', 
+           'resnet18', 
+           'resnet34', 
+           'resnet50', 
+           'resnet101', 
+           'resnet152', 
+           'VGG', 
+           'vgg11', 
+           'vgg13', 
+           'vgg16', 
+           'vgg19', 
+           'MobileNetV1', 
+           'mobilenet_v1', 
+           'MobileNetV2', 
+           'mobilenet_v2', 
+           'LeNet', 
+           'BaseTransform', 
```

Comment:
and transforms at `paddle.vision.transforms` level

Reply:
done